logos/src/logos/main.py

```
@@ -3575,12 +3575,27 @@ async def create_user(body: CreateUserRequest, request: Request):
+            body.prename, body.name, body.email, body.role
         )
+        if status == 200 and body.team_ids:
+            for team_id in body.team_ids:
+                db.add_team_member(team_id, user_dict["id"])
     if status != 200:
         raise HTTPException(status_code=status, detail=user_dict.get("error"))
     return {**user_dict, "logos_key": new_key}
 
+@app.post("/users/import", tags=["users"])
+async def import_users(file: UploadFile = File(...), request: Request = None):
+    logos_key = require_app_admin_or_above(request)
```

Flake8 (7.3.0)

[error] local variable 'logos_key' is assigned to but never used

(F841)